packages/@roots/bud-build/src/Build/index.ts

```diff
@@ -81,52 +75,60 @@ export class Build
    */
   @bind
   public async make(): Promise<Webpack.Configuration> {
-    await this.app.hooks.filterAsync<'event.build.make.before'>(
-      'event.build.make.before',
-      this.app,
-    )
+    try {
+      await this.app.hooks.filterAsync<'event.build.make.before'>(
+        'event.build.make.before',
+        this.app,
+      )
 
-    const build = await this.app.hooks.filterAsync<'build'>(
-      'build',
-    )
+      const build = await this.app.hooks.filterAsync<'build'>(
+        'build',
+      )
 
-    if (!build) {
-      throw new Error('Configuration could not be processed')
-    }
+      if (!build || isUndefined(build) || isNull(build)) {
+        throw new Error('bad build')
+      }
 
-    this.config =
-      await this.app.hooks.filterAsync<'event.build.override'>(
-        'event.build.override',
-        Object.entries(build).reduce(
-          (all: Configuration, [key, value]) => {
-            if (isUndefined(value) || isNull(value)) {
-              this.log(`warn`, {
-                message: `build.make: excluding ${key}`,
-                suffix: `value is undefined`,
-              })
-
-              return all
-            }
-
-            this.app.dump(value, {
-              prefix: `${chalk.bgBlue(
-                this.app.name,
-              )} config.${key}`,
-              maxDepth: 2,
+      const processed = Object.entries(build).reduce(
+        (all: Configuration, [key, value]) => {
+          if (!value || isUndefined(value) || isNull(value)) {
+            this.log(`warn`, {
+              message: `build.make: excluding ${key}`,
+              suffix: `value is undefined`,
             })
 
-            return {...all, [key]: value}
-          },
-          {},
-        ),
+            return all
+          }
+
+          this.log('info', key, prettyFormat(value))
+
+          return {...all, [key]: value}
+        },
+        {},
       )
 
-    await this.app.hooks.filterAsync<'event.build.make.after'>(
-      'event.build.make.after',
-      async () => null,
-    )
+      if (!processed.entry) processed.entry = {}
```

Comment:
:thinking: actually, this isn't great. need to think through how to differentiate between builds where an entrypoint should be left `undefined` vs. builds where an entrypoint should be an empty object.

- leaving it `undefined` means webpack will look for `src/index.js`
- setting it to empty obj means webpack won't

it might be just checking `app.hasChildren`, since this is the use case we are concerned with (multi-compiler parent compilations are designed to support zero entrypoints -- this is actually our "best practice").